src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_credentials/SshPrivateKeyCredential.java

```diff
@@ -30,6 +30,7 @@ public SshPrivateKeyCredential enterDirectly(String privateKey) {
 
     public Direct selectEnterDirectly() {
         WebElement e = choose("Enter directly");
+        find(by.input("./@class='secret-update-btn'")).click();
```

Comment:
```suggestion
        find(by.className("secret-update-btn")).click();
```

Simpler alternative
Seems better also in terms of code complexity ([fieldXPath](https://github.com/jenkinsci/acceptance-test-harness/blob/156754ea959583cd43b0a48681f5ac9f37b572ef/src/main/java/org/jenkinsci/test/acceptance/ByFactory.java#L166-L169))